### PR TITLE
Update to Laravel 8

### PR DESCRIPTION
**Pull request by HAV0C101**
#697 

Fixes #
Updates to Laravel 8

- [ ] Tested Changes using phpunit - see below
- [X] Have read and followed the Contribution Guidelines

Updated Laravel version and tidied up some code along the way
Only issue as of current is i cant get any of the unit tests to work they just clear the database as soon as i run them so no test data gets added and all the tests fail from all db query's returning null, i will keep working on this but i have also tested the whole app extensively beyond these tests as well

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -44,8 +49,3 @@ PUSHER_APP_CLUSTER=mt1
 
 MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
 MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
-
-SOLDER_REPO_LOCATION=/var/www/mods.solder.test/
```
Comment (phit) [inline .env.example:48]: where did these go?

Comment (HAV0C101) [reply .env.example:48]: oh my bad ill re add them, seems i missed a couple things

Comment (phit) [reply .env.example:48]: yeah there's a bunch of files also removed like the license

Comment (phit) [reply .env.example]: composer.json is also messed up

Comment (HAV0C101) [reply .env.example]: yeah ive just relised this, it sliped my mind, i just transfered over the actual app to a new laravel 8 instance

Comment (HAV0C101) [reply .env.example]: lemme fix all this real quick

Comment (phit) [reply .env.example]: no rush, just browsing the changed file list on Github

Comment (HAV0C101) [reply .env.example]: is there anything else you can see that i need to add back or fix?

Comment (HAV0C101) [reply .env.example]: thank you for going through this for me as well it helps to have a second eye to see all this

Comment (phit) [reply .env.example]: deleted:
```
.env.testing
.travis.yml
config/solder.php
phpspec.yml
storage/github-api-cache/.gitignore 
storage/debugbar/.gitignore
```

I think you also replaced the `public/favicon.ico`

and then the formatting in all the blade files is all over the place, probably wanna add this to the editorconfig and reformat them
```yaml
[*.blade.php]
indent_size = 2
```

Comment (phit) [reply .env.example]: actually I think they use tabs right now, and you used spaces

Comment (HAV0C101) [reply .env.example]: ah, ill re add those file and fix my indent errors, thanks for pointing all this out to me

**Comment by phit** [comment]
then with all the stylistic changes out of the way, why did you remove e.g. `laravelcollective/html` and redid all the forms?

**Comment by HAV0C101** [comment]
> then with all the stylistic changes out of the way, why did you remove e.g. `laravelcollective/html` and redid all the forms?

Seemed like a  more cleaner way of doing them to me and also removed an unnessacery dependency

**Pull request closed without merging**